Translation/settings.py

```diff
@@ -212,5 +206,4 @@
 PRINTED_DRAFT_TRANSLATIONS_ROOT = os.path.join(BASE_DIR, 'draft_translations/')
 HOST_URL = 'http://127.0.0.1:9000/'
 
-PRINT_ENABLED = PRINT_SYSTEM_ADDRESS is not None
-CUSTOM_PRINT_ENABLED = PRINT_ENABLED and False
+PRINT_ENABLED = True
```

Comment:
Let's remove this too.

Reply:
Ah... agree!

Reply:
Remove the `PRINT_ENABLED` ? But wouldn't it be used later on..?

Reply:
We can assume that it is always true.

Reply:
okay